src/format/table.rs

```diff
@@ -181,13 +183,24 @@ impl SeparatorBuilder {
 /// Struct for configuring a [`Table`](struct.Table.html)'s format
 #[derive(Debug, Default)]
 pub struct TableFormat {
+    pub(crate) foreground: Option<Color>,
     pub(crate) border: Border,
     pub(crate) separator: Separator,
 }
 
 impl TableFormat {
     /// Creates a new instance of [`TableFormat`](struct.TableFormat.html)
     pub fn new(border: Border, separator: Separator) -> Self {
-        Self { border, separator }
+        Self {
+            border,
+            separator,
+            foreground: None,
+        }
+    }
+
+    /// Set the foreground color of the table borders & separators.
+    pub fn foreground(mut self, color: Color) -> Self {
+        self.foreground = Some(color);
+        self
     }
```

Comment:
My Only Problem with this is first it must be assigned to a variable and then the foreground must be set. So Take this code:
```
Table::new(vec![], TableFormat::default().foreground(Color::Green));
```
it will fail because with this error
```
error[E0308]: mismatched types
  --> examples/simple.rs:33:9
   |
33 |         Table::new(vec![], TableFormat::default().foreground(Color::Red));
   |                            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected struct `cli_table::format::TableFormat`, found `&mut cli_table::format::TableFormat`

error: aborting due to previous error
```
But for consistency with SeparaterBuilder & BorderBuilder I could add a TableFormatBuilder struct. Though this seems like overkill for a struct with 3 fields. That would also fix the above problem